mlir/include/Quantum/Transforms/Passes.td

```diff
@@ -110,10 +110,10 @@ def RemoveChainedSelfInversePass : Pass<"remove-chained-self-inverse"> {
     let options = QuantumCircuitTransformationPass.options;
 }
 
-def MergeRotationPass : Pass<"merge-rotation"> {
+def MergeRotationsPass : Pass<"merge-rotations"> {
     let summary = "merge rotation boilerplate words";
 
-    let constructor = "catalyst::createMergeRotationPass()";
```

Comment:
Do we want to remove the fake boilerplate option as well?

Reply:
Do you want to keep it? I was thinking that having two passes is now sufficient for our testing

Reply:
I'm thinking we should have code that don't do anything, and since this option is not used in the pass I would prefer removing it.

Reply:
Ok so it is good, I prefer to remove it too